app/js/components/actions/actionsRule/actionsRule.controller.js

```diff
@@ -144,17 +144,16 @@ function ActionsRuleCtrl(
     $scope.plans = MaintenanceService.plans;
 
     $scope.showSystem = function (system) {
-        let systems;
+        const systems = RhaTelemetryActionsService.getClusterAffectedSystems();
 
-        if (typeof InsightsConfig.actionsShowSystem === 'function') {
-            return InsightsConfig.actionsShowSystem(system.toString());
-        }
-
-        systems = RhaTelemetryActionsService.getClusterAffectedSystems();
         if (typeof system === 'string' && systems && systems.hasOwnProperty(system)) {
             system = systems[system];
         }
 
+        if (typeof InsightsConfig.actionsShowSystem === 'function') {
+            return InsightsConfig.actionsShowSystem(system);
```

Comment:
Hey @lphiri do we really need to return a value here in this case? or can we just return after this call?
Down below this an empty return is performed otherwise, just asking.